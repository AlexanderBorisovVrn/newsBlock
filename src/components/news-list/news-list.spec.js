import React from 'react';
import NewsList from './news-list';
import {shallow} from 'enzyme';

const setUp =(props)=> shallow(<NewsList/>)

describe('render NewsList component', () => {
let comp;
beforeEach(()=>{comp = setUp()})
  it('should contain .news-list', () => {
    expect(comp).toMatchSnapshot();
  })

})