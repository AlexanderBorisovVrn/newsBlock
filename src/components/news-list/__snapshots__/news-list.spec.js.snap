// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render NewsList component should contain .news-list 1`] = `
<div
  className="news-list"
>
  <div
    className="news-list__inner"
  />
</div>
`;
